frame/staking/src/tests.rs

```diff
@@ -4711,3 +4711,32 @@ mod sorted_list_provider {
 		});
 	}
 }
+
+#[test]
+fn force_apply_min_commission_works() {
+	let prefs = |c| ValidatorPrefs { commission: Perbill::from_percent(c), blocked: false };
+	// let validators = || Validators::<Test>::iter().collect::<Vec<_>>();
+	ExtBuilder::default().build_and_execute(|| {
+		assert_ok!(Staking::validate(Origin::signed(30), prefs(10)));
+		assert_ok!(Staking::validate(Origin::signed(20), prefs(5)));
+
+		// // Given
+		// assert_eq!(validators(), vec![(31, prefs(10)), (21, prefs(5)), (11, prefs(0))]);
```

Comment:
commented code.

Reply:
🤦 fixed in [18c0089](https://github.com/paritytech/substrate/pull/10786/commits/18c00897d5e1aa17667f536f8fc5459a88699adf)